src/protocols/Http.ts

```diff
@@ -330,6 +359,9 @@ export default class HttpProtocol extends KuzzleAbstractProtocol {
 
       xhr.open(method, url);
 
+      // Authorize the reception of cookies
+      xhr.withCredentials = this.cookieSupport;
```

Comment:
Shouldn't this be set only when cookie auth is activated?

Reply:
This is the case when `cookieSupport` is enabled in the protocol, the cookieSupport of a protocol is only enabled when `cookieAuth` option is `true` in the Kuzzle constructor